### PR TITLE
feature(lwt basic longevity): LWT 3 hours longevity test

### PR DESCRIPTION
**Pull request by juliayakovlev**
## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I gave variables/functions meaningful self-explanatory names
- [x] I didn't leave commented-out/debugging code
- [x] I didn't copy-paste code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

New LWT 3 hours longevity:
with cassandra-stress user profile, run longevity test that:
1. insert data using "UPDATE .. IF.." syntax
2. run updates of non-PK columns using "UPDATE .. IF.." syntax
3. read the data

Run nemesis in parallel

If it will be approved, I could add data validation in the end of test.

[Jenkins job in the master](https://jenkins.scylladb.com/view/master/job/scylla-master/job/longevity/job/longevity-lwt-3h/)

### CODE REVIEW

File: data_dir/c-s_lwt_basic.yaml [hunk 1]
```diff
@@ -0,0 +1,72 @@
+# Based on https://gist.github.com/tjake/8995058fed11d9921e31
```
Comment (roydahan) [inline data_dir/c-s_lwt_basic.yaml]: what about using the same profile as Dmitry uses for performance?

**Comment by roydahan** [comment]
please add link to the jenkins job.

**Comment by juliayakovlev** [comment]
> please add link to the jenkins job.

Link in the descrition

File: data_dir/c-s_lwt_basic.yaml (hunk 1 above)
Comment (juliayakovlev) [reply data_dir/c-s_lwt_basic.yaml]: @roydahan  It's depend if we want to validate the results or not. 
I did user profile so, that we can to validate the results of updates

**Pull request merged**